src/screens/FeedScreen/RenderList.js

```diff
@@ -47,6 +48,9 @@ const RenderListFeed = (props) => {
     source = SOURCE_FEED_TAB,
     hideThreeDot = true,
     showAnonymousOption = false,
+    onDeletePost,
+    isShowDelete,
+    isSelf,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New props `onDeletePost`, `isShowDelete`, and `isSelf` have been added. Ensure that these props are passed correctly from the parent component and handled properly in this component.
```diff
+     onDeletePost,
+     isShowDelete,
+     isSelf,
      onHeaderOptionClicked = () => {}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->